src/primitives/mod.rs

```diff
@@ -21,7 +22,8 @@ impl From<AccountId> for LedgerJournalId {
         Self::from(uuid::Uuid::from(id))
     }
 }
-es_entity::entity_id! { ProfileId, PayoutQueueId, WalletId }
+
+es_entity::entity_id! { ProfileId, PayoutQueueId, WalletId, XpubId }
```

Comment:
@bodymindarts for now just to make the whole thing compile we added a XpubId. There also exists a XPubId which is a wrapper around fingerprint. curious to know your thoughts about this. Should we rename XPubId to something more appropriate ? XpubId was required because of es_entity